### PR TITLE
feat(database): add database movies migration

### PR DESCRIPTION
**Pull request by lvamaral**
What: created movies table and appropriate columns
Why: to prep our project to be able to read/write movies into the db

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,9 @@ node_js:
...
   - NODE_ENV="test"
+before_script:
+  - yarn run db:setup:user
+  - yarn run db:reset
 script:
-  - npm test
-  - npm run lint
- 
+  - yarn test
+  - yarn run lin
```
Comment (thetonywu) [inline .travis.yml]: typo

**Review by thetonywu — APPROVED** [review]
lgtm

File: db/seeds/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Movies = require('./data/movies');
+
+exports.seed = (Knex) => {
+  return Knex('movies').truncate()
+  .then(() => {
+    return Knex('movies').insert(Movies);
+  });
+};
```
Comment (virtualdom) [inline db/seeds/index.js]: When using ES6 arrow functions, it's really handy to shorthand functions that just return something, so in this case

```js
.then(() => {
  return Knex('movies').insert(Movies);
});
```
turns into 

```js
.then(() => Knex('movies').insert(Movies));
```

Technically we could shorthand the entire `exports.seed` function, but that doesn't do much for readability. For promise chains, though, using the arrow function shorthands makes things more readable in my opinion 😄

File: package.json [hunk 3]
```diff
@@ -23,7 +23,14 @@
...
     "test:raw": "NODE_ENV=test mocha test --require test/setup.js --recursive --timeout 30000",
-    "start": "nodemon --ignore test/"
+    "start": "nodemon --ignore test/",
+    "db:migrate": "knex migrate:latest --knexfile db/index.js",
+    "db:migrate:make": "knex migrate:make --knexfile db/index.js",
+    "db:reset": "if [ \"$NODE_ENV\" != \"production\" ]; then dropdb $(node -p \"require('./config').DB_NAME\"); yarn run db:setup && yarn run db:migrate; fi",
+    "db:rollback": "knex migrate:rollback --knexfile db/index.js",
+    "db:setup": "if [ \"$NODE_ENV\" != \"production\" ]; then createdb -O $(node -e \"var c = require('./config'); console.log(c.DB_USER, c.DB_NAME);\"); fi",
+    "db:setup:user": "if [ \"$NODE_ENV\" != \"production\" ]; then createuser $(node -p \"require('./config').DB_USER\"); fi",
+    "db:seed": "yarn run db:reset && knex seed:run --knexfile db/index.js"
```
Comment (virtualdom) [inline package.json]: nit: can you alphabetize the npm scripts? (looks like `enforce` and `start` are also in the wrong place, so maybe we should change that in Miyagi)

File: db/migrations/20180111112122_create_movies.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  return Knex.schema.createTable('movies', (table) => {
+    table.increments('id').primary();
+    table.text('title').notNullable();
+    table.integer('release_year');
+  });
+};
+
+exports.down = (Knex, Promise) =>  {
+  return Knex.schema.dropTable('movies');
+};
```
Comment (virtualdom) [inline db/migrations/20180111112122_create_movies.js]: I think this is a fair candidate for ES6 arrow shorthand too

```js
exports.down = (Knex) => Knex.schema.dropTable('movies');
```

File: db/migrations/20180111112122_create_movies.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+exports.up = (Knex, Promise) => {
```
Comment (virtualdom) [inline db/migrations/20180111112122_create_movies.js]: nit: since `Promise` isn't being used in this function, you can remove it!

**Review by virtualdom — CHANGES_REQUESTED** [review]
Just a handful of changes! Assign me again when you're ready!

File: db/seeds/index.js (hunk 2 above)
Comment (lvamaral) [reply db/seeds/index.js]: I agree! That part was copy-pasted from Miyagi so maybe we should change it there too.

File: .travis.yml (hunk 1 above)
Comment (virtualdom) [reply .travis.yml]: Should be `yarn run lint`!

**Review by virtualdom — APPROVED** [review]
LGTM once you fix the `yarn run lin` typo (if the build passes)

**Pull request merged**